docs/configuration/metrics/index.md

```diff
@@ -89,6 +89,7 @@ We also provide a simplified way to scrape the following Azure resources:
 - [Azure Service Bus Queue](service-bus-queue)
 - [Azure Storage Queue](storage-queue)
 - [Azure Virtual Machine](virtual-machine)
+- [Azure Cache for Redis](redis-cache)
```

Comment:
Nitpicking but can you move it to the top so that it's alphabetically correct, please?

Reply:
done.